util/index.ts

```diff
@@ -6,29 +14,29 @@ const { Order, Community } = require('../models');
 const logger = require('../logger');
 
 // ISO 4217, all ISO currency codes are 3 letters but users can trade shitcoins
-const isIso4217 = code => {
+const isIso4217 = (code: string): boolean => {
   if (code.length < 3 || code.length > 5) {
     return false;
   }
   const alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('');
-  code = code.toLowerCase().split('');
-  return code.every(letter => {
+  code = code.toLowerCase()
+  return code.split('').every(letter => {
     if (alphabet.indexOf(letter) == -1) {
       return false;
     }
     return true;
   });
 };
 
-const getCurrency = code => {
+const getCurrency = (code: string): (IFiat | boolean) => {
```

Comment:
@Mersho this looks like same bad practice as with validateParams? if yes, let's fix it in the same commit where we fix that

Reply:
@knocte Done. If there's anything wrong, let me know.

Reply:
It's done.